modules/gitops-configuring-user-level-access.adoc

```diff
@@ -28,15 +28,15 @@ metadata
     scopes: '[groups]'
 ----
 +
-. Add the `policy` configuration to the `rbac` section and add the `name`, `email` and the `role` of the user: 
+. Add the `policy` configuration to the `rbac` section and add the `name` and the desired `role` to be applied to the user:
```

Comment:
Hi @Dhruv-Soni11, could you please share the document you followed for this change

Reply:
Hey @varshab1210 - I made the changes for this from the following comment I have added in this PRs description.

Adding the comment here for reference:

This PR also includes the changes in content for correct policy for user level permissions from the following [PR](https://github.com/openshift/openshift-docs/pull/69135/files). Further details for this issue are available in the following: https://github.com/openshift/openshift-docs/issues/69134